dll/shellext/zipfldr/CZipCreator.cpp

```diff
@@ -65,7 +65,7 @@ DoGetNameInZip(const CStringW& basename, const CStringW& filename)
 
     ret.Replace(L'\\', L'/');
 
-    return DoGetAnsiName(ret);
+    return DoGetAnsiName(ret, CP_UTF8);
```

Comment:
This function is for getting ANSI name of the file, and this change effectively making it always return UTF-8 name instad. I think this is rather should be configurable by the user.

Reply:
...configurable by the user, or at least detected in run-time by checking file names to be packed.

Reply:
Configuration is deleted.